utils/manifests/generateComponent.go

```diff
@@ -20,7 +20,7 @@ func GenerateComponents(manifest string, resource int, cfg Config) (*Component,
 	wd := filepath.Join(utils.GetHome(), ".meshery", "bin")
 	fmt.Println("Looking for kubeopenapi-jsonschema in ", wd)
 	var binPath string = filepath.Join(wd, "kubeopenapi-jsonschema")
-	var url string = "https://github.com/layer5io/kubeopenapi-jsonschema/releases/download/v0.1.0/kubeopenapi-jsonschema"
+	var url string = "https://github.com/layer5io/kubeopenapi-jsonschema/releases/download/v0.1.2/kubeopenapi-jsonschema"
```

Comment:
we might want to fetch the latest release programmatically and not hard code it :sweat_smile:

Reply:
there is code for this in adapters which could be made quick work of, if we're willing to do so in this PR itself: https://github.com/meshery/meshery-istio/blob/master/internal/config/releases.go

Reply:
I have done this knowingly, to avoid breaking changes in meshery in case something incompatible takes place in the latest version.

Reply:
hmm... sure :+1:

Reply:
The issue is that we're not tracking versions here. This is off our radar. As the primary and really only intended consumers of this tool, we should (and will) only make releases when they are intended for this project's consumption.

Let's think aloud about possible approaches to both avoid bugs based on releases with breaking changes and not knowing that we've hardcoded a version here (out of the 60+ repos that we have with the thousands of dependencies that they have).

Reply:
Issue #110 created to follow up on this.